js/whitebit.js

```diff
@@ -610,17 +617,18 @@ module.exports = class whitebit extends Exchange {
             const success = this.safeValue (response, 'success');
             if (!success) {
                 const feedback = this.id + ' ' + body;
-                const message = this.safeValue (response, 'message');
-                if (typeof message === 'string') {
-                    const exact = this.safeValue (this.exceptions, 'exact', {});
-                    if (message in exact) {
-                        throw new exact[message] (feedback);
-                    }
-                    const broad = this.safeValue (this.exceptions, 'broad', {});
-                    const broadKey = this.findBroadlyMatchedKey (broad, message);
-                    if (broadKey !== undefined) {
-                        throw new broad[broadKey] (feedback);
-                    }
+                let message = this.safeValue (response, 'message');
+                if (typeof message !== 'string') {
+                    message = this.json (message);
```

Comment:
We have body (responseBody) as text, which is JSON already, so using `body` instead of `this.json` might improve it a bit ;)

Reply:
Yes, very good point! Thank you for fixing and fast merge of all PRs! :)

Reply:
@brandsimon appreciate your participation very much! tons of thx!